setup.py

```diff
@@ -27,13 +27,14 @@
     packages=find_namespace_packages(include=["dbt", "dbt.*"]),
     include_package_data=True,
     install_requires=[
-        "dbt-core>=1.2, <=1.4.5",
+        "dbt-core>=1.2, <=1.5.0",
```

Comment:
this might be too tight, dbt-core just released a minor patch v1.5.1 ;  and 1.5.2 is almost ready.
It might be better to change this to 
"dbt-core>=1.2, <1.6.0"

Reply:
Good point, changing that now. Thanks!